src/Symfony/Component/HttpKernel/DependencyInjection/Extension.php

```diff
@@ -20,10 +20,13 @@
  */
 abstract class Extension extends BaseExtension
 {
+    /** @var string[] */
```

Comment:
I don't think we need this type hint here. Static analysis is able to find it and it's a private property.

Reply:
Very well.